app/javascript/stylesheets/globalFooter.scss

```diff
@@ -0,0 +1,305 @@
+body {
```

Comment:
This is pure copypasta. There is probably a better way to do this? Or maybe we don't even need most of this? It's for the Stanford U. footer. Copied it out of EarthWorks.